sdk/cognitiveservices/azure-cognitiveservices-vision-face/azure/cognitiveservices/vision/face/version.py

```diff
@@ -9,5 +9,5 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "0.4.1"
+VERSION = "0.5.0"
```

Comment:
@lmazuel I'm not sure if this should be "0.5.0" or "0.4.2"

Reply:
Let's make it 0.5.0